### PR TITLE
Add k8s clusterrole to vmagent

### PR DESCRIPTION
**Pull request by afoninsky**

### CODE REVIEW

**Comment by tenmozes** [comment]
i will try to test this PR today and merge

File: charts/victoria-metrics-agent/templates/clusterrole.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```
Comment (tenmozes) [inline charts/victoria-metrics-agent/templates/clusterrole.yaml:1]: please add configuration to values.yaml to enable/disable creation of cluster role
e.g.
```
rbac:
  create: true
```
and use this param

File: charts/victoria-metrics-agent/templates/clusterrolebinding.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (tenmozes) [inline charts/victoria-metrics-agent/templates/clusterrolebinding.yaml:1]: as above

File: charts/victoria-metrics-agent/templates/clusterrole.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+kind: ClusterRole
+metadata:
```
Comment (tenmozes) [inline charts/victoria-metrics-agent/templates/clusterrole.yaml:3]: please add option to customize annotations

File: charts/victoria-metrics-agent/templates/clusterrolebinding.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+kind: ClusterRoleBinding
+metadata:
+  name: {{ template "chart.fullname" . }}-clusterrolebinding
```
Comment (tenmozes) [inline charts/victoria-metrics-agent/templates/clusterrolebinding.yaml:4]: please add option to customize annotations

**Comment by Denissa89** [comment]
Hello!
In my opinion, you should add nodes/proxy expansion to this RBAC.
The kubelet and cadvisor metrics are asked /api/v1/nodes/${1}/proxy/metrics/cadvisor and /api/v1/nodes/${1}/proxy/metrics.

**Comment by tenmozes** [comment]
thank you for contribution

**Pull request merged**